opm/autodiff/NewtonIterationUtilities.cpp

```diff
@@ -288,5 +289,24 @@ namespace Opm
 
 
 
+
+    /// Return true if this is a serial run, or rank zero on an MPI run.
+    bool isRankZero(const boost::any& parallel_info)
```

Comment:
In other classes we used `isIORank()` for this. Not sure whether we should/want to be consistent with the function names.